app/forms/routing/simulation_form.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Routing::Simulation
+class Routing::SimulationForm < ApplicationForm
```

Comment:
`Routing::SimulationForm#save!` should be changed to `#_save` according to `ApplicationForm` interface